pkg/testdata/prop-val-for-time-range-raw-data-for-time-range-A.golden.jsonc

```diff
@@ -16,15 +16,15 @@
 //  | Labels:                       | Labels:            | Labels:        |
 //  | Type: []time.Time             | Type: []float64    | Type: []string |
 //  +-------------------------------+--------------------+----------------+
-//  | 2021-02-01 17:30:00 +0100 CET | 28.287864119831497 | GOOD           |
-//  | 2021-02-01 17:35:00 +0100 CET | 28.345162289561085 | GOOD           |
-//  | 2021-02-01 17:40:00 +0100 CET | 28.57531512865568  | GOOD           |
-//  | 2021-02-01 17:45:00 +0100 CET | 28.779631112505346 | GOOD           |
-//  | 2021-02-01 17:50:00 +0100 CET | 28.746434495312684 | GOOD           |
-//  | 2021-02-01 17:55:00 +0100 CET | 28.815794249589597 | GOOD           |
-//  | 2021-02-01 18:00:00 +0100 CET | 29.67006840640403  | GOOD           |
-//  | 2021-02-01 18:05:00 +0100 CET | 29.504278603194113 | GOOD           |
-//  | 2021-02-01 18:10:00 +0100 CET | 29.32301703873501  | GOOD           |
+//  | 2021-02-01 08:30:00 -0800 PST | 28.287864119831497 | GOOD           |
```

Comment:
> Is this just because the original "golden" files were generated in CET and you are coming from PST?

yep, it auto updates when i run the tests from vscode.

> I'm wondering if we really master these golden files. Are they supposed to exist so that in the future we are aware of any output changes? (understand this is probably out of scope of this PR)

yea, think they're supposed to make it easier to compare outputs so we don't have to handwrite the outputs to use for assertions.